nodeServerless/Local CE SAM/CE_lambda_functions/card_get.js

```diff
@@ -0,0 +1,62 @@
+/**
+ *
+ * Event doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html#api-gateway-simple-proxy-for-lambda-input-format
+ * @param {Object} event - API Gateway Lambda Proxy Input Format
+ *
+ * Context doc: https://docs.aws.amazon.com/lambda/latest/dg/nodejs-prog-model-context.html 
+ * @param {Object} context
+ *
+ * Return doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html
+ * @returns {Object} object - API Gateway Lambda Proxy Output Format
+ * 
+ */
+
+let response;
+var AWS = require('aws-sdk');
+AWS.config.update({
+    region: "us-west-2"
+})
+
+let docClient = new AWS.DynamoDB.DocumentClient();
+// let ddb = new AWS.DynamoDB();
+
+exports.lambdaHandler = async (event, context) => {
+    try {
+        console.log("\n\n*********************\n");
+        var id = event.queryStringParameters.scryfallid;
+        var name = event.queryStringParameters.name;
+
+        console.log("id: " + id + ", type: " + typeof id + ". name: " + name + ", type: " + typeof name);
+        
+        var params = {
+            TableName: "BulkMagicCards",
+            Key: {
+                'scryfallid' : id,
+                'name' : name
+            }      
+        };
+
+        try {
+            const data = await docClient.get(params).promise();
+            return { statusCode: 200, body: JSON.stringify(data) };
+        } catch (error) {
+            return {
+              statusCode: 400,
```

Comment:
4xx is user error, 5xx is server error.
https://en.wikipedia.org/wiki/List_of_HTTP_status_codes